sqrt/__init__.py

```diff
@@ -48,6 +49,8 @@ def get_data_stream(iterable):
 
 
 def main(save_to, num_batches, continue_=False):
+    if continue_:
+        continue_training(save_to)
```

Comment:
I suggest we simple get rid of this `continue_` flag and remove its usage for the test. `sqrt.py` used to be the test for dumping, but now since it is in another repo, it should not play such a role any more.